test/units/TEST-17-UDEV.rename-netif.sh

```diff
@@ -174,7 +174,7 @@ EOF
     test -n "$found"
 
     journalctl --sync
-    timeout -v 30 journalctl _PID=1 _COMM=systemd --since "$since" -n all --follow | grep -m 1 -q -F 'foobar: systemd-udevd failed to process the device, ignoring: File exists'
```

Comment:
as in https://github.com/systemd/systemd/pull/39937 this really needs to be fixed in musl or its polyfills, rather than being worked around everywhere else, as the latter just won't scale.